client/components/mma/cancel/cancellationSaves/ContinueMembershipConfirmation.tsx

```diff
@@ -37,19 +43,19 @@ export const ContinueMembershipConfirmation = () => {
 				`}
 			/>
 			<Stack space={4}>
-				<Heading>Thank you for continuing your membership</Heading>
+				<h2 css={headingCss}>Thank you for keeping your Membership</h2>
 				<p>
-					No action is needed from your side. Your membership will
-					continue as normal.
+					The new price of your Membership is £7/month. <br />
```

Comment:
Very small nitpick: it looks like the line break is only on desktop screen sizes per the designs

Reply:
Can you suggest a way of doing this? The way I'm thinking of seems a bit clunky for one line break

Reply:
Sorted after chatting offline